src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/CreateInitialDataSetVersionFunctionTests.cs

```diff
@@ -46,6 +49,7 @@ public async Task Success()
 
             await AddTestData<ContentDbContext>(context =>
             {
+                context.ReleaseVersions.Add(releaseVersion);
```

Comment:
Does the saving of the ReleaseFiles not cascade save the ReleaseVersions that they're attached to?  Just wondering why you need to explicitly add an `Add` here, not that it's a biggie or owt :)

Reply:
Yes it should do.

I split these out in an attempt to fix some flaky tests, but it made no difference so I've undone the explicit `Add` of the release version now, thanks.